tests/test_functionals.py

```diff
@@ -1,5 +1,5 @@
 import unittest2 as unittest
-from casacore import functionals
+from pyrap.functionals import *
 
```

Comment:
pyrap is the old deprecated name for python-casacore, please use casacore instead.